src/qt/forms/optionsdialog.ui

```diff
@@ -216,6 +216,33 @@
          </property>
         </widget>
        </item>
+       <item>
+        <layout class="QHBoxLayout" name="horizontalLayout_3_Display">
+         <item>
+          <widget class="QLabel" name="thirdPartyTxUrlsLabel">
+           <property name="toolTip">
+            <string>Third-party URLs (e.g. a block explorer) that appear in the transactions tab as context menu items. %s in the URL is replaced by transaction hash. Multiple URLs are separated by vertical bar |.</string>
```

Comment:
(change in both the tooltips if taken)
```suggestion
            <string>Third-party URLs (e.g. a block explorer) that appear in the transactions tab as context menu items. %s in the URL is replaced by the transaction hash. Multiple URLs are separated by a vertical bar |.</string>
```
(Feel free to ignore: I wonder if `%s` would be better as `"%s"` to help it not appear to be a bug like `"Dear {$customer_name}"`)

Reply:
Looks like while doing this you could add a missing hyphen to "third party tool" in `src/qt/forms/optionsdialog.ui:192`.